engine/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@
 # > cmake -DCMAKE_BUILD_TYPE=Debug <path to source tree>
 #====================================
 
-CMAKE_MINIMUM_REQUIRED(VERSION 2.6)
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8.8)
```

Comment:
Is this supported on all our platforms? I think we kept it at 2.6 for Ubuntu 14.04.

Reply:
I bumped it up because `PYTHONLIBS_VERSION_STRING` (which was already in use prior to this PR) requires at least 2.8.8, according to the relevant CMake [documentation page](https://cmake.org/cmake/help/latest/module/FindPythonLibs.html).

A general overview of CMake versions across various linux/bsd distributions (that have it in their repositories) can be seen [here](https://repology.org/badge/vertical-allrepos/cmake.svg?minversion=2.8.8) (all bright red are below 2.8.8). As to MacOS or Windows... well that's more up to the user.

As to Ubuntu 14.04, specifically:
* According to [repology](https://repology.org/project/cmake/versions), 14.04 "Trusty" has two versions available: 2.8.12 and 3.5.1 (trusty/main and trusty-updates respectively)
  - (The official Ubuntu packages site no longer lists Trusty)
  - (Note that 12.04 - the previous LTS - has 2.8.7)
* I was under the impression that this project had officially dropped support for 14.04 in #90?

Reply:
Nice to see you here @s0600204 glad you have the time to help.

Reply:
@s0600204 I was noting the history. I'd love to bump it up to 3 even, just have to be aware of supporting platforms. If 2.8.8 or later is on all our supported platforms then great, let's bump it up.

Thanks!!!